yoga/model/helpers.py

```diff
@@ -1,11 +1,61 @@
 from ._assimp import ffi
 
 import io
+import re
 import os.path
+import unidecode
 import yoga.image
 
 
+def normalize_path(path):
+    # Expects a unicode path, returns a ascii one.
+    # Paths are normalized to a standard linux relative path,
+    # without a point, and lowercase.
+    # That is to say /images\subfolder/..\texture.png -> images/texture.png
+    # It does not correspond to an effective path,
+    # as the backslashes on linux are wrongly seen as separators.
+    # This function is meant to give a standard output.
+
+    path = unidecode.unidecode(path)
+    split_path = re.findall(r"[\w\s\-_.:]+", path)
+    normalized_path = ""
+    ignored_folders = 0
+
+    for i, name in enumerate(reversed(split_path)):
```

Comment:
the reverse can be done using slices:
`reversed(split_path)` → `split_path[::-1]`

you can also doing it earlier like this:

```python
splited_path = path.replace("\\", "/").split("/")[::-1]
```